commands/trajectoryauto.py

```diff
@@ -0,0 +1,28 @@
+from commands2 import SequentialCommandGroup
+from wpimath.geometry import Pose2d, Rotation2d, Translation2d
+from wpimath.trajectory import TrajectoryGenerator, TrajectoryConfig
+from commands.followtrajectory import FollowTrajectory
+from commands.resetdrive import ResetDrive
+
+from subsystems.drivesubsystem import DriveSubsystem
+import constants
+
+
+class TrajectoryAuto(SequentialCommandGroup):
+    def __init__(self, drive: DriveSubsystem) -> None:
+        self.drive = drive
+        trajectoryConfig = TrajectoryConfig(
+            constants.kMaxForwardLinearVelocity, constants.kMaxForwardLinearAcceleration
+        )
+        trajectoryConfig.setKinematics(self.drive.kinematics)
+
+        trajectory = TrajectoryGenerator.generateTrajectory(
+            Pose2d(0, 0, Rotation2d(0)),
```

Comment:
Are these relative to the robot? If not, how do these numbers work/how were they selected (basically I'm saying I don't understand how this trajectory is generated)?

Reply:
coordinates in relation to robot odometry